mlir/test/Dialect/Bufferization/Transforms/finalizing-bufferize.mlir

```diff
@@ -63,7 +63,7 @@ func.func @fancy_layout_to_no_layout_cast(%m: memref<?xf32, strided<[100], offse
 //  CHECK-SAME:     %[[arg:.*]]: memref<?xf32, strided<[1], offset: 25>>)
 //       CHECK:   %[[c0:.*]] = arith.constant 0 : index
 //       CHECK:   %[[dim:.*]] = memref.dim %[[arg]], %[[c0]]
-//       CHECK:   %[[alloc:.*]] = memref.alloc(%[[dim]]) : memref<?xf32>
+//       CHECK:   %[[alloc:.*]] = memref.alloc(%[[dim]]) {{.*}} : memref<?xf32>
 //       CHECK:   memref.copy %[[arg]], %[[alloc]]
```

Comment:
I guess the default option here is `memref.copy` and that is why none of the tests have a change although you use the option now to decide for which copy function to use? I vaguely remember there were some issues with testing other option configurations, is that still the case? Otherwise, it would be nice to test that.

Reply:
Yeah, none of the passes called here have pass options that allow to influence the copyFn in the BufferizationOptions.